plugins/postcss-design-tokens/README.md

```diff
@@ -0,0 +1,276 @@
+		"background": {
+			"primary": { "value": "#fff" }
+		}
+	}
+}
+```
+
+```json
+{
+	"color": {
+		"background": {
+			"primary": { "value": "#000" }
+		}
+	}
+}
+```
+
+**And this CSS :**
+
+```pcss
+@design-tokens url('./tokens-light.json') format('style-dictionary3');
+@design-tokens url('./tokens-dark.json') when('dark') format('style-dictionary3');
+
+.foo {
+	color: design-token('color.background.primary');
+}
+```
+
+**You can configure :**
+
+##### No `is` option.
+
+```js
+postcssDesignTokens()
+```
+
+```pcss
+@design-tokens url('./tokens-light.json') format('style-dictionary3');
+@design-tokens url('./tokens-dark.json') when('dark') format('style-dictionary3');
+
+.foo {
+	color: design-token('color.background.primary');
+}
+
+/* becomes */
+
+.foo {
+	color: #fff;
+}
+```
+
+##### `is` option set to 'dark'.
+
+```js
+postcssDesignTokens({ is: ['dark'] })
+```
+
+```pcss
+@design-tokens url('./tokens-light.json') format('style-dictionary3');
+@design-tokens url('./tokens-dark.json') when('dark') format('style-dictionary3');
+
+.foo {
+	color: design-token('color.background.primary');
+}
+
+/* becomes */
+
+.foo {
+	color: #000;
+}
+```
+
+### unitsAndValues
+
+The `unitsAndValues` option allows you to control some aspects of how design values are converted to CSS.
+`rem` <-> `px` for example can only be calculated when we know the root font size.
+
+#### rootFontSize
+
+defaults to `16`
+
+```js
+postcssDesignTokens({
+	unitsAndValues: {
+		rootFontSize: 20,
+	},
+})
+```
+
+```pcss
+@design-tokens url('./tokens.json') format('style-dictionary3');
+
+.foo {
+	color: design-token('color.background.primary');
+	padding-top: design-token('size.spacing.small');
+	padding-left: design-token('size.spacing.small' to px);
+	padding-bottom: design-token('size.spacing.small' to rem);
+}
+
+/* becomes */
+
+.foo {
+	color: #fff;
+	padding-top: 16px;
+	padding-left: 16px;
+	padding-bottom: 0.8rem;
+}
+```
+
+## Syntax
+
+[PostCSS Design Tokens] is non-standard and is not part of any official CSS Specification.
+
+### `@design-tokens` rule
+
+The `@design-tokens` rule is used to import design tokens from a JSON file into your CSS.
+
+```pcss
+@design-tokens url('./tokens.json') format('style-dictionary3');
+```
+
+```pcss
+@design-tokens url('./tokens.json') format('style-dictionary3');
+@design-tokens url('./tokens-dark-mode.json') format('style-dictionary3') when('dark');
+```
+
+```
+@design-tokens [ <url> | <string> ]
+               [ when(<theme-condition>*) ]?
+               format(<format-name>);
+
+<theme-condition> = <string>
+
+<format-name> = [ 'style-dictionary3' ]
+```
+
+All `@design-tokens` rules in a document are evaluated in order of appearance.
+If a token with the same path and name already exists it will be overridden.
+
+All `@design-tokens` rules are evaluated before any `design-token()` functions.
+
+`@design-tokens` rules can be conditional through `when` conditions. Multiple values can be specified in `when`.
```

Comment:
Should we add a comment that specifies that when multiple it means `AND`, not `OR`? Or something as in, all conditions must be met?

Reply:
Added this bit : 
> Multiple conditions always have an `AND` relationship.